File: 6/Problem5.c
#include<stdio.h>

void TableReverse(int iNo)
{
    int iCnt = 0;
    int iT = 0;
    for(iCnt = 10; iCnt >= 1; iCnt--)
    {
        iT = iNo * iCnt;
        printf("%d\n",iT);
    }
    
    
}
int main()
{
    int iValue = 0;
     printf("Enter the number : ");
     scanf("%d",&iValue);

     TableReverse(iValue);

     return 0;
}

File: 21/problem5.java
import java.util.*;
class DigitX
{
    public static void DisplayDivisible(int iNo)
    {
        int iCnt = 0;
        for( iCnt = 1; iCnt <= iNo; iCnt++)
        {
            if((iCnt % 2 == 0) && (iCnt % 3 == 0))

            {

                System.out.print(iCnt + " ");

            }
        }
    }
}


class problem5
{
    public static void main(String args[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.println("Enter the number : ");
        iValue = sobj.nextInt();

        DigitX.DisplayDivisible(iValue);
    }
}